### ISSUE TITLE
#378102910

### ISSUE THREAD

**Issue by fycee** [bitshares/bitshares-ui#2828]
Buying BTS using my 2,555.6 Weku Coins was not yet filled. Thank you for being prompt with my issue the last time. I hope that this will be fixed the soonest as well. Have a great day!

**Comment by sschiessl-bcp** [comment]
An order must be matched by someone else that wants to sell BTS for that price. If you can't match at that price, try adjusting your price to current market rates.

**Comment by fycee** [comment]
I even created a blog out of this process. I used the latest price of
WEKU_BTS that time.

Check it out!

https://deals.weku.io/community-deals/@fycee/buying-bts-using-my-1-880-wekucoin-earnings-from-my-blogs


Anyway, I cancelled that order already and created a new one. Order ID #
378455637
[image: image.png]

[image: image.png]

I used 0.06 BTS latest price.
The difference from yesterday's price if it was filled and been exchanged
to USD to what I should get now is around 3 USD. Anyway, I am a very
understanding person.

Thank you and I hope you'll be able to fix this issue.
